Week3/HW3/3_moreJavascript/9_functionVehicle.js

```diff
@@ -0,0 +1,12 @@
+let vehicle = ["tram","metro"];
+let newVehicle = ["motorbike", "caravan", "bike"];
+vehicle.push(...newVehicle);
+
+function vehicleType(color, code, age) {
+let type = vehicle[code];
+    ( age > 2) ? condition = "used" : condition = "new" ;
```

Comment:
Same comment about conditional operators as added in exercise `6_vehicleAge.js` 🙂

Reply:
Thank you for your time.
I corrected it, too, sir.